js/app.jsx

```diff
@@ -486,6 +486,25 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      if (!str) {
+        return null;
+      }
+      var linesRegex = /(\w+)\.scala:(\d+)/i;
```

Comment:
Just because we're expecting to process Scala files here. I guess since we're only showing the scalding.step.descriptions, we can probably forgo the function and just make it specific to processing scalding steps.